tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -1126,6 +1165,7 @@ std::uint64_t ROOT::Experimental::Internal::RNTupleFileWriter::RFileProper::Writ
    RTFString strObject;
    RTFString strTitle;
    RTFKey keyHeader(offset, offset, strClass, strObject, strTitle, len, nbytes);
+   keyHeader.MakeBigKey();
```

Comment:
Consider adding a comment why this is a unconditional.

Reply:
Ok.  Can  you remind me why we don't know whether the key will land past the 2GB line (TKey seems to be able to know)?

Reply:
We use `TKey::Create()` to reserve the space for the key in the file. To find the space, `Create()` already needs to know the length of the key and the big key provides a key length that always works. I think `TBasket` does the same, i.e. it is unconditionally a big key.

Reply:
You are completely right. I had forgotten https://github.com/root-project/root/commit/3970c0beada6d13d4087dd7ac34def7aba390887.